ci/libbuild.sh

```diff
@@ -1,9 +1,9 @@
 #!/usr/bin/bash
 
 pkg_upgrade() {
-    echo "Running yum -y distro-sync... $(date)"
-    yum -y distro-sync
-    echo "Done yum -y distro-sync! $(date)"
+    echo "Running dnf -y distro-sync... $(date)"
```

Comment:
I know this will be an endless battle but...we probably do want to support rhel8 builds at some point, and it's still `yum` there...not that it matters really.  I just find it easier personally to type `yum` since it works on rhel7 too.